pygeoapi/api/__init__.py

```diff
@@ -885,60 +882,85 @@ def describe_collections(api: API, request: APIRequest,
     else:
         collections_dict = collections
 
-    LOGGER.debug('Processing parameter-name parameter')
-    parameternames = request.params.get('parameter-name') or []
-    ext_qargs = (f'?parameter-name={parameternames}'
-                 if isinstance(parameternames, str) else '')
-    if isinstance(parameternames, str):
-        parameternames = parameternames.split(',')
-        onto_mapping = get_mapping(parameternames)
-    else:
-        onto_mapping = get_mapping()
+    # LOGGER.debug('Processing parameter-name parameter')
+    # parameternames = request.params.get('parameter-name') or []
+    # ext_qargs = (f'?parameter-name={parameternames}'
+    #              if isinstance(parameternames, str) else '')
+    # if isinstance(parameternames, str):
+    #     parameternames = parameternames.split(',')
```

Comment:
```suggestion
    if isinstance(parameternames, str):
        parameternames = parameternames.split(',')
```

Parameter names still needs to be parsed into a list